package.json

```diff
@@ -61,7 +61,8 @@
   },
   "scripts": {
     "atlas": "node ./test/atlas_connectivity_tests.js",
-    "test": "npm run lint && mongodb-test-runner -t 60000 test/core test/unit test/functional",
+    "orchestration": "node test/check_mongo_orchestration_tests.js",
+    "test": "npm run orchestration && npm run lint && mongodb-test-runner -t 60000 test/core test/unit test/functional",
```

Comment:
If `check_mongo_orchestration_tests.js` is just echoing the location of `orchestration`, why do we need to run it as part of the `test` script?

Reply:
I wasn't sure if we wanted this to run automatically when running tests. I can remove it though!